api/docs.go

```diff
@@ -20,6 +20,49 @@ const docTemplate = `{
     "host": "{{.Host}}",
     "basePath": "{{.BasePath}}",
     "paths": {
+        "/popular_repositories/": {
+            "get": {
+                "description": "Get popular repositories",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "tags": [
+                    "popular",
+                    "repositories"
+                ],
+                "summary": "List Popular Repositories",
+                "operationId": "listPopularRepositories",
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/api.PopularRepositoriesCollectionResponse"
+                        }
+                    },
+                    "400": {
+                        "description": "Bad Request",
+                        "schema": {
+                            "$ref": "#/definitions/errors.ErrorResponse"
+                        }
+                    },
+                    "404": {
+                        "description": "Not Found",
+                        "schema": {
+                            "$ref": "#/definitions/errors.ErrorResponse"
+                        }
+                    },
+                    "500": {
+                        "description": "Internal Server Error",
+                        "schema": {
+                            "$ref": "#/definitions/errors.ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
         "/repositories/": {
             "get": {
                 "description": "list repositories",
```

Comment:
I'd get more specific here, now that there are specific repository types.  Rhetorically, does this include popular repositories?

Not sure about the value of going all the way to rename this custom_repositories, but at least this description should indicate a distinction.